jax/_src/numpy/lax_numpy.py

```diff
@@ -4067,6 +4067,25 @@ def polyint(p, m=1, k=None):
     return true_divide(concatenate((p, k)), coeff)
 
 
+@_wraps(np.polydiv)
+def polydiv(u,v):
+  _check_arraylike("polydiv", u, v)
+  u, v = _promote_dtypes_inexact(u, v)
+
+  w = u[0] + v[0]
+  m = len(u) - 1
+  n = len(v) - 1
+  scale = 1. / v[0]
```

Comment:
Some major care still needs to be taken here, since in compiled code we generally don't have the guarantee that `v` is leading-zero-free.

Reply:
oh okay just saw